### PR TITLE
Optimize Vec3 use constructor default.

### PR DESCRIPTION
**Pull request by perminovVS**
Make Vec3 vec3(Vec3::ZERO);   to   Vec3 vec3;

extansion #11288

### CODE REVIEW

File: getsuitablefopen.patch [hunk 1]
```diff
@@ -0,0 +1,792 @@
+﻿From 19780399aae4e77419dbd4e89f135784bfba7c6c Mon Sep 17 00:00:00 2001
```
Comment (minggo) [inline getsuitablefopen.patch]: Why uploaded this patch file?

**Comment by minggo** [comment]
Could you please fix conflict and fix compiling error.

**Comment by minggo** [comment]
Could you please fix conflict?

**Comment by perminovVS** [comment]
@minggo yes of course

**Comment by perminovVS** [comment]
fixed all in #11518

**Pull request closed without merging**